diff --git a/test/test_jsc/test_ejs_integration.py b/test/test_jsc/test_ejs_integration.py
index 798481079..2a03304f7 100644
--- a/test/test_jsc/test_ejs_integration.py
+++ b/test/test_jsc/test_ejs_integration.py
@@ -7,6 +7,7 @@
 
 import pytest
 
+from yt_dlp_plugins.extractor.ytjsc import AppleWebKitJCP
 from yt_dlp.extractor.youtube.jsc.provider import (
     JsChallengeRequest,
     JsChallengeType,
@@ -103,7 +104,7 @@ def url(self, /):
     responses.append(JsChallengeProviderResponse(request, JsChallengeResponse(test.type, output_type(test.values))))
 
 
-@pytest.fixture(params=[BunJCP, DenoJCP, NodeJCP, QuickJSJCP])
+@pytest.fixture(params=[BunJCP, DenoJCP, NodeJCP, QuickJSJCP, AppleWebKitJCP])
 def jcp(request, ie, logger):
     obj = request.param(ie, logger, None)
     if not obj.is_available():
